src/endless/EndlessUsbTool.cpp

```diff
@@ -68,6 +69,7 @@ BOOL CEndlessUsbToolApp::InitInstance()
 	InitCommonControlsEx(&InitCtrls);
 
 	CWinApp::InitInstance();
+	CWinApp::SetRegistryKey(_T("Endless"));
```

Comment:
I see the registry key is made by adding this vendor to the app name, EndlessUsbTool... if we change that to Endless Installer will that break the heck out of everything / resources / etc @raduancuta ?